### PR TITLE
upgrades latest gradle and build tools

### PR DESCRIPTION
**Pull request by lkwg82**
I double checked the DeviceFragment Crash in the Emulator and on my cm13 android device. It is gone!

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,15 +46,7 @@ android {
...
-    // Without this parameter, all builds fail with error:
-    //     Execution failed for task ':mergeArmeabiDebugResources'.
-    //     > Some file crunching failed, see logs for details
-    // https://code.google.com/p/android/issues/detail?id=163378
-    //
-    // NOTE: Setting this causes a crash when opening the DeviceFragment:
-    //       Resources$NotFoundException: File res/drawable/ic_tag_text_outline_black_24dp.xml
-    //       This means we can't use build tools 2.0 until this is fixed.
-    //aaptOptions.setProperty("cruncherEnabled", false)
+    aaptOptions.setProperty("cruncherEnabled", false)
```
Comment (Nutomic) [inline build.gradle]: Did you check if this is still needed? If yes, you should leave that part of the comment.

File: gradle.properties [hunk 2]
```diff
@@ -1 +1,2 @@
 android.useDeprecatedNdk=true
+org.gradle.jvmargs=-Xmx2048M
```
Comment (Nutomic) [inline gradle.properties]: Why are you setting this?

Comment (lkwg82) [inline gradle.properties]: this follows a warning by the gradle run ... see the console output without it.

File: build.gradle (hunk 1 above)
Comment (lkwg82) [inline build.gradle]: Still needed and yes comments should be left in place.

**Pull request merged**